src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -167,7 +168,18 @@ def update_argument(self, param_name, argtype):
     def __call__(self, *args, **kwargs):
         if isinstance(self.command_source, ExtensionCommandSource) and self.command_source.overrides_command:
             logger.warning(self.command_source.get_command_warn_msg())
-        return super(AzCliCommand, self).__call__(*args, **kwargs)
+
+        cmd_args = args[0]
```

Comment:
is it certain that args will have at least one element?

Reply:
Yes. It is the same in knack.